tests/lib/c_lib/src/main.c

```diff
@@ -631,6 +815,13 @@ void test_main(void)
 			 ztest_unit_test(test_bsearch),
 			 ztest_unit_test(test_abs),
 			 ztest_unit_test(test_atoi),
+			 ztest_unit_test(test_snprintf),
+			 ztest_unit_test(test_fprintf),
+			 ztest_unit_test(test_vfprintf),
+			 ztest_unit_test(test_vprintf),
```

Comment:
For these testcases, can we add to suitable directory? Maybe it is more appropriate to put testcases in directory of `tests/lib/sprintf/`.

Reply:
ok , I moved these testcases to teset/lib/sprintf/